invenio_communities/generators.py

```diff
@@ -275,6 +276,9 @@ def excludes(self, member_types=None, **kwargs):
         """Preventing needs."""
         if member_types:
             for m in member_types:
+                if m=='group' and \
```

Comment:
is there any way to avoid hardcoding `group` in this condition statement ? ping @ntarocco

Reply:
+1 to this

Reply:
we discussed this with @Pineirin, when called, it is passing a string `group`... so the `AllowedMemberTypes` is already quite coupled with `user`/`email`/`group` strings.
Another way is to create a totally new permission class (as he did before), e.g. `VerifyGroupEnabled` or whatever, but if you override `can_members_add` in your instance and you don't add that class, then it will not work.
One or the other...

Reply:
I don't think it would make sense to use a configurable value in the generator and not in the permission itself. I would either use types defined in the configuration for all AllowedMemberTypes. In case two, I'm ok with it but I don't know if the con Nicola mentioned is too big.

I don't have enough confidence to state that any solution better (as Nicola said, I've been bouncing between both).

Reply:
It was decided IRL, that we would use the second solution.